xdocs/index.xml

```diff
@@ -70,73 +70,53 @@ list of changes.
       </ul>
 
     <p>
-      In debian based Linux those dependencies could be installed by something like:
+        For Debian-based Linux distributions, these dependencies can be installed using the following command:
     </p>
-    <source>apt-get install libapr1.0-dev libssl-dev</source>
+    <source>apt-get install libapr1-dev libssl-dev openjdk-11-jdk</source>
```

Comment:
Tomcat native doesn't require Java 11, Java 8 will do fine.

Reply:
Judging by the [README.txt](https://github.com/apache/tomcat-native/blob/main/README.txt#L36) that states Java 11 is required for building, I chose to recommend openjdk-11-jdk package. Shall I change it?

Reply:
Indeed, you are right. My mad.